src/bin/zmrabbitmqctl

```diff
@@ -0,0 +1,174 @@
+#!/bin/sh
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2016, 2021 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+# 
+NAME=rabbitmq-server
+DAEMON=/opt/zimbra/common/sbin/${NAME}
+CONTROL=/opt/zimbra/common/sbin/rabbitmqctl
+DESC=rabbitmq
+USER=zimbra
+PID_FILE=/opt/zimbra/data/rabbitmq/pid
```

Comment:
I thought pid files are stored in /opt/zimbra/log directory

Reply:
yes correct